README.md

```diff
@@ -292,32 +379,29 @@ Restoring a `builder` database is exactly like restoring any other database--whi
         `/hab/pkgs/core/postgresql/<version>/<release>/bin/pg_restore --host=<url_of_pg_host> --dbname=builder builder.dump`
 1. Start the on-prem Builder services
 
-        * In some cases your version of Postgres might not have a createdb binary in which case you'll want to connect to database to run the create db command
+    > note: In some cases your version of Postgres might not have a createdb binary in which case you'll want to connect to database to run the create db command.
```

Comment:
Capitalize "note"?

Reply:
Addressed in https://github.com/habitat-sh/on-prem-builder/pull/212/commits/38454a460a0f673bf5f0cde08392dfcb3e36b1f6